examples/web-wagmi/src/misc/LoginSpecificProfile.tsx

```diff
@@ -56,16 +58,15 @@ function AuthenticatedContent({
 
   const onLoginClick = async () => {
     await logout();
-
     if (isConnected) {
       await disconnectAsync();
     }
 
     const { connector } = await connectAsync();
 
     if (connector instanceof InjectedConnector) {
-      const signer = await connector.getSigner();
-      await login(signer, handle);
+      const publicClient = getPublicClient();
+      await login(ethersProviderFromPublicClient(publicClient).getSigner(), handle);
```

Comment:
we still use this check to determine whether there is a connector or not, as it can be `undefined`

although we only support the `InjectedConnector` in this example app so maybe we could just check it is defined